documentation/istio_keycloak_integration.asciidoc

```diff
@@ -0,0 +1,213 @@
+:toc: macro
+toc::[]
+:idprefix:
+:idseparator: -
+== Authentication with Keycloak and Istio
+
+=== Introducing _https://www.keycloak.org/[Keycloak]_
```

Comment:
Keep the titles as simple as possible to allow bookmarking and deep-linking
https://github.com/devonfw/docgen/blob/master/documentation/guidelines.asciidoc

So simply use `Introduction` or `Introducing Keycloak`.
Then put the link in the text instead where you start with `Keycloak is ...`.
Otherwise we will run into issues with docgen.